.gitignore

```diff
@@ -1,5 +1,5 @@
 ## Ignore WebConfig because it contains sensitive values
-Web.config
+Authy2FA/Web.config
```

Comment:
Please have a look at 84208e9, this uses the `Local.config` you suggested. BTW, nice post.